site/content/en/docs/Advanced/high-availability-agones.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "High Availability Agones"
+date: 2023-02-08T15:02:23Z
+weight: 20
+description: >
+  Allows controllers to be strictly for control loops; feature separates extensions and webhooks out of your controllers
+---
+
+## High Availability for Agones Controller
+
+The `agones-controller` pod is currently not only used for control loops of game server related resources, but also include other services such as webhooks and allocation extension srvices. High availability agones mean as opposed to having controller host everything mentioned, we will add additional `extensions` pods that exposes the webhook and allocations service. Thus, making `agones-controller` pod strictly for control loops. Read more about pod disruption [here](https://agones.dev/site/docs/advanced/controlling-disruption/)
```

Comment:
The wording here is awkward. Taking a stab at better wording:

---

When `SplitControllerAndExtensions` is enabled, the `agones-controller` responsibility is split up into `agones-controller`, which enacts the Agones control loop, and `agones-extensions`, which acts as a service endpoint for webhooks and the allocation extension API. Splitting these responsibilities allows the `agones-extensions` pod to be horizontally scaled, making the Agones control plane highly available.

---

I would drop the reference to disruption as that link is really talking about gameserver pod disruption.